homeassistant/components/image_processing/__init__.py

```diff
@@ -28,6 +28,11 @@
 
 SCAN_INTERVAL = timedelta(seconds=10)
 
+DEVICE_CLASSES = [
+    'alpr',        # licence plate
```

Comment:
OK, that makes sense. In that case can you extend the comment to read `# automatic license plate recognition`. That way it clarifies this refers to license plates in general, not this specific platform.